docker-envs/ror-with-elastic-cloud-poc/elastic-cloud/docker-compose.yml

```diff
@@ -0,0 +1,156 @@
+    ulimits:
+      memlock:
+        soft: -1
+        hard: -1
+
+  es02:
+    build:
+      context: .
+      dockerfile: es/Dockerfile
+      args:
+        ELK_VERSION: $ELK_VERSION
+        ELASTIC_PASSWORD: $ELASTIC_PASSWORD
+    ports:
+      - "5002:58672"
+      - "9302:9300"
+    environment:
+      - cluster.name=es-cluster
+      - node.name=es02-master
+      - node.roles=master,data
+      - bootstrap.memory_lock=true
+      - discovery.seed_hosts=es01,es03
+      - cluster.initial_master_nodes=es01-master,es02-master,es03-master
+      - xpack.security.enabled=true
+      - xpack.security.transport.ssl.enabled=true
+      - xpack.security.transport.ssl.verification_mode=none
+      - xpack.security.transport.ssl.client_authentication=none
+      - xpack.security.transport.ssl.keystore.path=elastic-certificates.p12
+      - xpack.security.transport.ssl.truststore.path=elastic-certificates.p12
+      - xpack.security.http.ssl.enabled=true
+      - xpack.security.http.ssl.verification_mode=none
+      - xpack.security.http.ssl.client_authentication=none
+      - xpack.security.http.ssl.keystore.path=elastic-certificates.p12
+      - xpack.security.http.ssl.truststore.path=elastic-certificates.p12
+      - "ES_JAVA_OPTS=-Xms512m -Xmx512m -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:58672"
+    networks:
+      - es-network
+    ulimits:
+      memlock:
+        soft: -1
+        hard: -1
+
+  es03:
+    build:
+      context: .
+      dockerfile: es/Dockerfile
+      args:
+        ELK_VERSION: $ELK_VERSION
+        ELASTIC_PASSWORD: $ELASTIC_PASSWORD
+    ports:
+      - "5003:58672"
+      - "9303:9300"
+    environment:
+      - cluster.name=es-cluster
+      - node.name=es03-master
+      - node.roles=master,data
+      - bootstrap.memory_lock=true
+      - discovery.seed_hosts=es01,es02
+      - cluster.initial_master_nodes=es01-master,es02-master,es03-master
+      - xpack.security.enabled=true
+      - xpack.security.transport.ssl.enabled=true
+      - xpack.security.transport.ssl.verification_mode=none
+      - xpack.security.transport.ssl.client_authentication=none
+      - xpack.security.transport.ssl.keystore.path=elastic-certificates.p12
+      - xpack.security.transport.ssl.truststore.path=elastic-certificates.p12
+      - xpack.security.http.ssl.enabled=true
+      - xpack.security.http.ssl.verification_mode=none
+      - xpack.security.http.ssl.client_authentication=none
+      - xpack.security.http.ssl.keystore.path=elastic-certificates.p12
+      - xpack.security.http.ssl.truststore.path=elastic-certificates.p12
+      - "ES_JAVA_OPTS=-Xms512m -Xmx512m -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:58672"
+    networks:
+      - es-network
+    ulimits:
+      memlock:
+        soft: -1
+        hard: -1
+
+  es-xpack-proxy:
+    image: mitmproxy/mitmproxy
+    command:
+      mitmdump --set validate_inbound_headers=false --mode reverse:https://es02:9200 --ssl-insecure --verbose
```

Comment:
Why we need a rev proxy?

Reply:
we don't need it but it's useful to sniff communication between KBN and ES

Reply:
Ok, I do this stuff as well in local testing with a node is snippet sometimes 👍